test/snapshots/main.js.md

```diff
@@ -33,6 +33,7 @@ Generated by [AVA](https://avajs.dev).
         },
         env: {},
         id: 'sapper',
+        logo: undefined,
```

Comment:
I'm curious why `logo` is being added here only for Sapper when the PR is adding only Gatsby logos at the moment.

Reply:
It's being added here because with the overall set of changes `logo` is being returned for all frameworks as part of the response body. 

This particular snapshot is for a different test, and uses sapper instead of Gatsby, which is why this is appearing